docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md

```diff
@@ -51,59 +51,184 @@ The following example shows how to create a NebulaGraph cluster by creating a cl
+                  zone_list: az1,az2,az3
+            ```
+
+            **Operator's Use of Zone Information**
+
+            NebulaGraph Operator leverages Kubernetes' [TopoloySpread](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/) feature to manage the scheduling of Storage and Graph pods. Once the `zone_list` is configured, Storage services are automatically assigned to their respective Zones based on the `topology.kubernetes.io/zone` label.
+
+            For intra-zone data access, the Graph service dynamically assigns itself to a Zone using the `--assigned_zone=$NODE_ZONE` parameter. It identifies the Zone name of the node where the Graph service resides by utilizing an init-container to fetch this information. The Alpine Linux image specified in `spec.alpineImage` (default: `reg.vesoft-inc.com/nebula-alpine:latest`) plays a role in obtaining Zone information.
+
+            **Prioritizing Intra-Zone Data Access**
+
+            By setting `spec.graphd.config.prioritize_intra_zone_reading` to `true` in the cluster configuration file, you enable the Graph service to prioritize sending queries to Storage services within the same Zone. In the event of a read failure within that Zone, the behavior depends on the value of `spec.graphd.config.stick_to_intra_zone_on_failure`. If set to `true`, the Graph service avoids reading data from other Zones and returns an error. Otherwise, it reads data from leader partition replicas in other Zones.
+
+            ```yaml
+            spec:
+              alpineImage: reg.vesoft-inc.com/cloud-dev/nebula-alpine:latest
+              graphd:
+                config:
+                  prioritize_intra_zone_reading: "true"
+                  stick_to_intra_zone_on_failure: "true"
+            ```
+
+            **Zone Mapping for Resilience**
+
+            Once Storage and Graph services are assigned to Zones, the mapping between the pod and its corresponding Zone is stored in a configmap named `nebula-graphd|storaged-zone`. This mapping facilitates pod scheduling during rolling updates and pod restarts, ensuring that services return to their original Zones as needed.      
+
+
+        ??? info "Expand to view part of the sample cluster configurations"
+
+            ```yaml
+            apiVersion: apps.nebula-graph.io/v1alpha1
+            kind: NebulaCluster
+            metadata:
+              name: nebula
+              namespace: default
+            spec:
+              # Alpine image built with Linux tools.
+              alpineImage: "reg.vesoft-inc.com/nebula-alpine:latest"
+              graphd:
+                config:
+                  ## Prioritize sending queries to the storage nodes in the same zone.
+                  prioritize_intra_zone_reading: "true"
+                  # Stick to intra-zone routing if unable to
+                  # find the requested part in the zone when set to true.
+                  stick_to_intra_zone_on_failure: "true"
```

Comment:
Can you set `stick_to_intra_zone_on_failure` to false here, as it's the value we expect users to use in the current version. Don't give users illusions.